tests/printf_float/main.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2016 Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       print floating point values test application
+ *
+ * This test is supposed to check that floating point values can be
+ * displayed with printf function.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+
+int main(void)
+{
+    double floating_point_value = 2016.0317;
+    char result[8] = {0};
```

Comment:
This does not really mean what it seems like it does. See http://stackoverflow.com/questions/201101/how-to-initialize-an-array-in-c